### ISSUE TITLE
cannot use a special query term more than once

### ISSUE THREAD

**Issue by mziwisky** [livegrep/livegrep#247]
Maybe I just can't figure out the syntax I should use, but it seems like it's not supported to do something like `some_term -repo:foo -repo:bar`, is that correct?

**Comment by nelhage** [comment]
That's correct; this isn't currently supported. The workaround is to enable regular expression mode, and do `-repo:(foo|bar)`. There's no reason the tool couldn't do that itself internally, though, so I'll leave this open to track that feature request.

**Comment by mziwisky** [comment]
Ah, thanks for the workaround
